examples/complete/outputs.tf

```diff
@@ -1,14 +1,11 @@
-output "id" {
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
we prob don't need these comments b/c it's obvious that the file is for the outputs, and the comments just consume space

```
# ---------------------------------------------------------------------------------------------------------------------
# MODULE OUTPUTS
# ---------------------------------------------------------------------------------------------------------------------
```

Reply:
b8e0620d6187eb3896e25ea4609ea65b09a8ee24